config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  resources :products
```

Comment:
you can do here:
```ruby
resources :products, only: %i[index new create]
```

and usually it's a good practice to add `root` before `resources`